boards/nrf51dk/src/main.rs

```diff
@@ -243,6 +245,13 @@ pub unsafe fn reset_handler() {
                          12);
     virtual_alarm1.set_client(timer);
 
+    let radio_virtual_alarm = static_init!(
+        VirtualMuxAlarm<'static, Rtc>,
+        VirtualMuxAlarm::new(mux_alarm),
+        24);
```

Comment:
Replace this with a `192/8`. When you get a `static_init!` size wrong, the compiler complains and tells you how many bits it ought to be, so leaving the size in units bits makes it easier to fix up when necessary.